### ISSUE TITLE
RP8 has a formatting issue

### ISSUE THREAD

**Issue by mbaudin47** [mbaudin47/otbenchmark#18]
The notebook RP8 has a formatting issue:

https://github.com/JEBROUN/otbenchmark/blob/master/examples/RP8.ipynb

![image](https://user-images.githubusercontent.com/31351465/81092853-84902580-8f01-11ea-958d-351f7a659a70.png)

@JEBROUN

**Comment by mbaudin47** [comment]
@JEBROUN : could you please fix this formatting issue?

**Comment by mbaudin47** [comment]
@JEBROUN : did you see this issue?

**Comment by JEBROUN** [comment]
#25

**Comment by mbaudin47** [comment]
Thanks @JEBROUN . Do you know that you can write your commit messages so that the corresponding issues are automatically closed?

This is documented here:

https://help.github.com/en/enterprise/2.16/user/github/managing-your-work-on-github/closing-issues-using-keywords

Hence, I suggest that the message of a commit which fixes one specific issue should look like this:

```
ReliabilityProblem8: Fix notebook equation format
<empty blank line>
Closes #18
```

ie dont mention the issue # in the commit title, but in the body where "Closes" allows to automatically close it upon merge on github.

I hope that this will help you to use github more easily.

**Comment by mbaudin47** [comment]
The problem is fixed, this is why I close this issue.
